packages/admin-app/src/pages/ElectionListPage.tsx

```diff
@@ -1,6 +1,115 @@
-import { Grid } from '@material-ui/core';
-import React from 'react';
+import { ApiClientFactory, Election } from '@electionguard/api-client';
+import { Grid, Container, makeStyles } from '@material-ui/core';
+import { DataGrid, GridColDef } from '@material-ui/data-grid';
+import React, { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import FilterToolbar from '../components/FilterToolbar';
+import GoHomeButton from '../components/GoHomeButton';
+import InternationalText from '../components/InternationalText';
 
-export const ElectionListPage: React.FC = () => <Grid>Election List</Grid>;
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 0.75,
+    },
+    title: {
+        fontSize: 40,
+        textAlign: 'center',
+        display: 'block',
+        paddingBottom: theme.spacing(2),
+    },
+    content: {
+        paddingTop: theme.spacing(4),
+        paddingBottom: theme.spacing(4),
+    },
+    grid: {
+        minHeight: 200,
+    },
+    electionState: {
+        textTransform: 'lowercase',
+    },
+    navArea: {
+        paddingBottom: theme.spacing(4),
+        textAlign: 'center',
+    },
+}));
+
+export const ElectionListPage: React.FC = () => {
+    const initialElections: Election[] = [];
+    const [elections, setElections] = useState(initialElections);
+    const [textResult, setTextResult] = useState('');
+    const [pageSize, setPageSize] = React.useState(10);
+    const classes = useStyles();
+    const intl = useIntl();
+
+    const columns = (): GridColDef[] => [
+        {
+            field: 'election_id',
+            headerName: intl.formatMessage({
+                id: 'election_list_page.election_id_header',
+            }),
+            width: 300,
+            headerClassName: 'bold-style--header',
+        },
+        {
+            field: 'key_name',
+            headerName: intl.formatMessage({
+                id: 'election_list_page.key_name_header',
+            }),
+            width: 300,
+            headerClassName: 'bold-style--header',
+        },
+        {
+            field: 'state',
+            headerName: intl.formatMessage({
+                id: 'election_list_page.state_header',
+            }),
+            width: 150,
+            headerClassName: 'bold-style--header',
+            cellClassName: classes.electionState,
+        },
+    ];
+
+    const getElections = async () => {
+        const service = ApiClientFactory.getMediatorApiClient();
+        try {
+            const electionResults = await service.findElection({}, 0, 100);
+            if (electionResults) {
+                setElections(electionResults);
+            }
+        } catch (ex) {
+            setTextResult('An error occurred');
+        }
+    };
+    useEffect(() => {
+        getElections();
+    }, []);
+    return (
+        <Grid container className={classes.root}>
+            <Container maxWidth="md" className={classes.content}>
```

Comment:
This top level container we should use at the Layout level not the page level.

Reply:
Agreed. I'm going to defer implementing this until a subsequent PR to avoid merge conflicts.

Reply:
https://github.com/microsoft/electionguard-ui/issues/107